artifacts/ts/Greeter.ts

```diff
@@ -56,6 +64,39 @@ export namespace GreeterTypes {
 }
 
 class Factory extends ContractFactory<GreeterInstance, GreeterTypes.Fields> {
+  readonly defaultInitialFields: GreeterTypes.Fields = {
+    btcPrice: BigInt(0),
+    array0: [
+      [BigInt(0), BigInt(0)],
+      [BigInt(0), BigInt(0)],
+      [BigInt(0), BigInt(0)],
+    ],
+    array1: [
+      [false, false],
+      [false, false],
+      [false, false],
+    ],
+    array2: [
+      ["", ""],
+      ["", ""],
+      ["", ""],
+    ],
+    array3: [
+      [
+        "tgx7VNFoP9DJiFMFgXXtafQZkUvyEdDHT9ryamHJYrjq",
+        "tgx7VNFoP9DJiFMFgXXtafQZkUvyEdDHT9ryamHJYrjq",
+      ],
+      [
+        "tgx7VNFoP9DJiFMFgXXtafQZkUvyEdDHT9ryamHJYrjq",
+        "tgx7VNFoP9DJiFMFgXXtafQZkUvyEdDHT9ryamHJYrjq",
+      ],
+      [
+        "tgx7VNFoP9DJiFMFgXXtafQZkUvyEdDHT9ryamHJYrjq",
+        "tgx7VNFoP9DJiFMFgXXtafQZkUvyEdDHT9ryamHJYrjq",
+      ],
+    ],
+  };
```

Comment:
This would generate a lot of code. Is it possible to do it this way:
```
function getDefaultInitialFields(): GreeterTypes.Fields = {
  // call a shared function from contract.ts
}
```